tools/data_gen/src/main.rs

```diff
@@ -0,0 +1,197 @@
+#[macro_use]
+extern crate lazy_static;
+
+use argh::FromArgs;
+use calyx::{errors::CalyxResult, frontend, ir};
+use serde::ser::{Serialize, Serializer};
+use serde_json::{json, Map, Value};
+use std::borrow::Borrow;
```

Comment:
you definitely 5000% _do NOT_ want this imported. It's going to shadow the borrow method on `RefCell` and generally make a mess of things. The borrow trait is unneeded here.